dashboard-for-web/articles/web-dashboard-designer-mode/provide-data/working-with-sql-data-sources/pass-query-parameters.md

```diff
@@ -4,7 +4,7 @@ author: Natalia Kazakova
 legacyId: 117969
 ---
 # Pass Query Parameters
-The [Query Builder](../query-builder.md) allows you to [filter queries](filter-queries.md) using parameters. To specify settings of an added query parameter after creating a query, click **Next** in the [Dashboard Data Source Wizard](../dashboard-data-source-wizard.md) dialog.
+The [Query Builder](../../ui-elements/dialogs-and-wizards/query-builder.md) allows you to [filter queries](filter-queries.md) using parameters. To specify settings of an added query parameter after creating a query, click **Next** in the [Dashboard Data Source Wizard](../../ui-elements/dialogs-and-wizards/dashboard-data-source-wizard.md) dialog.
 
```

Comment:
Last sentence in Line 18: For example, you can use an existing [dashboard parameter](../../data-analysis/dashboard-parameters.md) to pass to the SQL query.

Can we say the following instead?

`For example, you can pass an existing [dashboard parameter](../../data-analysis/dashboard-parameters.md) to the SQL query.`